src/components/SQForm/SQFormCheckboxGroupItem.tsx

```diff
@@ -58,19 +46,23 @@ function SQFormCheckboxGroupItem({
     return !!field.value;
   }, [value, field]);
 
+  const theme = useTheme();
   return (
     <FormControlLabel
-      className={`
-        ${classes.checkboxGroupItem}
-        ${isRowDisplay ? classes.rowDisplay : ''}
-      `}
+      sx={[
```

Comment:
This prop will actually take a function who's first param is the theme. So we _shouldn't_ need to use `useTheme`

```
sx={(theme: Theme) => { return {/* some css */}}}

Reply:
@20BBrown14 Have refactored to use the `theme` prop from sx.  Thanks